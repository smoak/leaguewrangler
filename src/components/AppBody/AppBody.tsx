import React, { SFC } from 'react';

import Routes from '../Routes';

const AppBody: SFC = () => {
  return (
    <div className="AppBody">
      <div className="AppBody__main">
        <Routes />
      </div>
    </div>
  );
};

export default AppBody;
