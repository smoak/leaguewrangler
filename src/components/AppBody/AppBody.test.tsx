import { shallow, ShallowWrapper } from 'enzyme';
import AppBody from './AppBody';
import React from 'react';

describe('AppBody', () => {
  let component: ShallowWrapper;

  beforeEach(() => {
    component = shallow(<AppBody />);
  });

  it('renders', () => {
    expect(component).toMatchSnapshot();
  });
});
