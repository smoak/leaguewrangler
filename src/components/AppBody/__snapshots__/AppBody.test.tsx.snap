// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppBody renders 1`] = `
<div
  className="AppBody"
>
  <div
    className="AppBody__main"
  >
    <Routes />
  </div>
</div>
`;
